packages/blocks/src/surface-block/element-model/shape.ts

```diff
@@ -112,11 +113,11 @@ export class ShapeElementModel extends GfxPrimitiveElementModel<ShapeProps> {
     return 'shape';
   }
 
-  @yfield('#000000')
-  accessor color!: string;
+  @yfield()
+  accessor color: string | CustomColor = '#000000';
```

Comment:
This change will lead AFFiNE preloading page throw a error. May be there are some legacy data of `shape` do not  contains `color`

![CleanShot 2024-07-26 at 22 19 55@2x](https://github.com/user-attachments/assets/33970a8a-235d-41b8-8f45-df697fb4c676)

Reply:
Good catch. I will check that.

Reply:
But it should return `#00000`.